pygmt/tests/test_datasets_earth_vertical_gravity_gradient.py

```diff
@@ -0,0 +1,96 @@
+"""
+Test basic functionality for loading Earth vertical gravity gradient datasets.
+"""
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pygmt.datasets import load_earth_vertical_gravity_gradient
+from pygmt.exceptions import GMTInvalidInput
+
+
+def test_earth_vgg_fails():
```

Comment:
Also need to update the test names from `test_earth_vgg_` to `test_earth_vertical_gravity_gradient_`